### DISCUSSION TITLE
Traceback (most recent call last) & PermissionError

### DISCUSSION THREAD

**Discussion by viv1010** [Atri-Labs/atrilabs-engine#657] [Help: Installation & Start]
![Capture 2](https://user-images.githubusercontent.com/104632737/217433988-9102f389-c301-4935-837a-8aaf19a3c6ee.JPG)

**Comment by cruxcode** [comment]
This is new. Can you check what is the permission for C:\\Users\\Ganesh\\.atri\\webapp-builder-win.exe?

**Comment by viv1010** [comment]
All the permissions are enabled.
